server/import.go

```diff
@@ -32,7 +32,33 @@ func CreateImportBlob(ekPub crypto.PublicKey, sensitive []byte, pcrs *tpmpb.Pcrs
 	private := createPrivate(sensitive, ek.NameAlg)
 	public := createPublic(private, ek.NameAlg, pcrs)
 
+	return createImportBlobHelper(ek, public, private, pcrs)
+}
+
+// CreateSigningKeyImportBlob uses the provided public EK to encrypt the signing
```

Comment:
Like `CreateImportBlob`, we should mention the actual client-side command that can be used `tpm2tools Key.ImportSigningKey`.